backend/src/main/scala/com/softwaremill/bootzooka/util/Hashed.scala

```diff
@@ -0,0 +1,3 @@
+package com.softwaremill.bootzooka.util
+
+trait Hashed
```

Comment:
sorry, I have to change my previous suggestion ;) `PasswordHash` was better, as hashes are different, and a password hash has different properties than other hashes.